globalMotionPlannerROS:

  ## Particle Swarm Optimization(PSO) planner 
  # number of particles
  n_particles: 50
  # number of inherited particles (Note: Need to be less than parameter n_ Particles)
  n_inherited: 20
  # number of position points contained in each particle
  pointNum: 5
  # The maximum velocity of particle motion
  max_speed: 40
  # inertia weight
  w_inertial: 1.0
  # social weight
  w_social: 2.0
  # cognitive weight
  w_cognitive: 1.2
  # obstacle factor(greater means obstacles)
  obs_factor: 0.39       
  # Set the generation mode for the initial position points of the particle swarm
  # 1: Randomly generate initial positions of particle swarm within the map range
  # 2: Randomly generate initial particle swarm positions within the circular area of the starting and target points
  initposmode: 2
  # Whether to publish particles
  pub_particles: false
  # maximum iterations
  pso_max_iter: 5

